docker/Dockerfile.devel

```diff
@@ -66,12 +67,17 @@ WORKDIR /dali
 
 COPY . .
 
+ARG TRITON_VERSION
```

Comment:
Do you need this? Doesn't tritonserver have its own env var with the version?

Reply:
Probably it has, but here we are building completely outside tritonserver. After the build we're deploying the build result to the tritonserver. Therefore we need to match the version of what we are building with were we are deploying